website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+
+## Overview of Mesh
+
+<detailsToggle alt_header="How do I implement contracts for my models?">
+
+In dbt, [model contracts](/docs/collaborate/govern/model-contracts) serve as a governance tool enabling the definition and enforcement of data structure standards in your dbt models. They allow you to specify and uphold data model guarantees, including column data types, allowing for stability of dependent models. Should a model fail to adhere to its established contracts, it will not successfully build.
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are model versions?">
+
+dbt [model versions](https://docs.getdbt.com/docs/collaborate/govern/model-versions) are iterations of your dbt models made over time. In many cases, you may knowingly choose to change a model’s structure in a way that “breaks” the previous model contract, and may break downstream queries depending on that model’s structure. When you do so, you may find it useful to create a new version of the model to signify this change.
+
+You can use **model versions** to:
+
+- Test "prerelease" changes (in production, in downstream systems)
+- Bump the latest version, to be used as the canonical source of truth
+- Offer a migration window off the "old" version
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are model access modifiers?">
+
+A [model access modifier](/docs/collaborate/govern/model-access) in dbt determines if a model is accessible as an input to other dbt models and projects. It specifies where a model can be referenced using [the `ref` function](https://docs.getdbt.com/reference/dbt-jinja-functions/ref). There are three types of access modifiers:
+
+1. **Private:** A model with a private access modifier is only referenceable by models within the same group. This is intended for models that are implementation details and are meant to be used only within a specific group of related models.
+2. **Protected:** Models with a protected access modifier can be referenced by any other model within the same dbt project or when the project is installed as a package. This is the default setting for all models, ensuring backward compatibility, especially when groups are assigned to an existing set of models.
+3. **Public:** A public model can be referenced across different groups, packages, or projects. This is suitable for stable and mature models that serve as interfaces for other teams or projects.
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are model groups?">
+
+A [model group](/docs/collaborate/govern/model-access#groups) in dbt is a concept used to organize models under a common category or ownership. This categorization can be based on various criteria, such as the team responsible for the models or the specific data source they model.
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are the main benefits of implementing dbt Mesh?">
+
```

Comment:
Should we add an intro sentence so this doesn't go directly into a numbered list?